tests/analysis/chemenv/coordination_environments/test_read_write.py

```diff
@@ -127,8 +127,6 @@ def test_structure_environments_neighbors_sets(self):
             str(nb_set) == "Neighbors Set for site #6 :\n - Coordination number : 4\n - Voronoi indices : 1, 4, 5, 6\n"
         )
 
-        assert nb_set == nb_set
```

Comment:
It's so weird, there are multiple comparison with itself in 0ea568e48b9d0dbff7a8c39c43c048db41898500, does this serve any purpose that I'm not aware of?

Reply:
yes, these are there to check correctness of custom `__eq__` methods

Reply:
it might make sense to ignore `PLR0124` specifically in test files rather than ignore this error on a case by case basis

Reply:
Thanks, I was about to add it to `[tool.ruff.lint.per-file-ignores]` too :)